operator/flags.go

```diff
@@ -35,6 +35,10 @@ func init() {
 		"Subnets IDs (separated by commas)")
 	option.BindEnv(operatorOption.IPAMSubnetsIDs)
 
+	flags.StringToStringVar(&operatorOption.Config.IPAMInstancesTags, operatorOption.IPAMInstancesTags, operatorOption.Config.IPAMInstancesTags,
+		"EC2 Instances tags in the form of k1=v1,k2=v2 (multiple k/v pairs can also be passed by repeating the CLI flag")
+	option.BindEnv(operatorOption.IPAMInstancesTags)
```

Comment:
I'm noticing that we are inconsistent between `Instances tags` and `Instance tags`. I propose we go with the latter everywhere including the flag name as well.

Reply:
yeah, I tried to reflect the `subnet(s) tag(s)` naming, but I'll change it to all use the same as `Instance tags`